apps/console/src/features/applications/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -314,12 +316,28 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
      * Handles the update button click event.
      */
     const handleUpdateClick = (): void => {
-        eventPublisher.publish("application-sign-in-method-click-update-button");
-        
         if (AdaptiveScriptUtils.isEmptyScript(adaptiveScript)) {
             setAdaptiveScript(AdaptiveScriptUtils.generateScript(steps + 1).join("\n"));
             setIsDefaultScript(true);
         }
+
+        const eventPublisherProperties = {
```

Comment:
Lets type `eventPublisherProperties`.
```suggestion
        const eventPublisherProperties: { "script-based": boolean;  "step-based": <TYPE_OF_STEP> } = {
```

Reply:
Fixed